### PR TITLE
New branch "for_review"

### PR DESCRIPTION
**Pull request by Azonkeu**
This pull request is to implement the new changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <body>
+
+    <!-- <header> -->
+      <nav id="navbar">
+        <a class="nav" href="#" >  About</a>
+        <a  class="nav" id="profile-link"  href="https://github.com/Azonkeu" target="_blank">  Work</a>
+        <a class="nav" href="#" >  Contact</a>
+
+
+      </nav>
```
Comment (Azonkeu) [inline index.html:18]: Please do not add blank lines, spaces, or indentations without a reason.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+
+    <!-- <header> -->
+      <nav id="navbar">
+        <a class="nav" href="#" >  About</a>
+        <a  class="nav" id="profile-link"  href="https://github.com/Azonkeu" target="_blank">  Work</a>
+        <a class="nav" href="#" >  Contact</a>
```
Comment (Azonkeu) [inline index.html:15]: Please use a `ul` with `li`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        </section>
```
Comment (Azonkeu) [inline index.html:38]: Please use a list `ul` with `li`.

**Review by Azonkeu — COMMENTED** [review]
# Required Changes ♻
## Hi Ornela Azonkeu Macheteu 👋,
### You have been doing a tremendous job so far 😀

# To Highlight 
GitHub flow is followed. ✔
PR has a title and description. ✔

# To Improve 🙏
Please check inline comments under the review

# Optional suggestions
Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better.

# Happy coding! 👍

Please leave any questions or comments in the PR if you have any doubts or unclarity!
Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html (hunk 2 above)
Comment (wandji20) [reply index.html:15]: - [ ] Great catch and nice proposition. However, including a reason for requiring this change will nice

File: index.html (hunk 3 above)
Comment (wandji20) [reply index.html:38]: - [ ] Kindly add a reason for this change so the developer may understand why they need to make this change

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @Azonkeu  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  giving reviews for your past project.
However, there is one issue you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Use of emojis :100: 
:heavy_check_mark: Review summary :100: 
:heavy_check_mark: Polite comments :100: 

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Azonkeu) [inline index.html:15]: Please use a `ul` with `li`. It is a great way to  organize your site's navigation and the semantic markup will improve accessibility for users that require screen-readers.

File: index.html (hunk 1 above)
Comment (Azonkeu) [inline index.html:18]: Please do not add blank lines, spaces, or indentations without a reason.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        <h2 class="project-tile">These are some of my projects</h2>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
```
Comment (Azonkeu) [inline index.html:37]: Please use a `ul` with `li`. The semantic markup will improve accessibility for users that require screen-readers.

**Review by Azonkeu — COMMENTED** [review]
# Required Changes ♻

## Hi Ornela Azonkeu Macheteu 👋,
### You have been doing a tremendous job so far 👏👏 

# To Highlight
GitHub flow is followed. ✔
PR has a title and description. ✔

# To Improve 💪
Please check inline comments under the review

# Optional suggestions
Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better.

# Happy coding! 👏👏👍

**Please leave any questions or comments in the PR if you have any doubts or unclarity!**

**Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Review by bimbolabuari — COMMENTED** [review]
# Project Approved 🚀

Hi @Azonkeu 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**